src/main/java/com/eternalcode/discordapp/filter/FilterResult.java

```diff
@@ -5,14 +5,10 @@ public final class FilterResult {
     private static final FilterResult PASSED = new FilterResult(true);
     private static final FilterResult NOT_PASSED = new FilterResult(false);
 
-    private final boolean passed;
+    private final boolean isPassed;
```

Comment:
Zostałbym przy nazwie passed.
```suggestion
    private final boolean passed;
```